MSFragger-GUI/src/com/dmtavt/fragpipe/cmd/CmdMSBooster.java

```diff
@@ -143,14 +144,6 @@ public boolean configure(Component comp,
       return false;
     }
 
-    boolean hasTimsTof = false;
-    for (InputLcmsFile t : lcmsToFraggerPepxml.keySet()) {
-      if (t.getPath().getFileName().toString().endsWith(".d")) {
-        hasTimsTof = true;
-        break;
-      }
-    }
-
```

Comment:
Why removing this check?

Reply:
I tested with an uncalibrated mzml produced by msfragger from .d folder and IM was always turned off, even though MSBooster could still extract IM data from the mzml. MSBooster also turns off useIM if it cannot find IM data for a scan

Reply:
The uncalibrated mzml is not supposed to be used as the FragPipe input. It is more like an intemediate file.

We also not recommend converting .d to mzML.

Reply:
added back in latest commit